release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3137,7 +3163,7 @@ In the following tables, features are marked with the following statuses:
 |Cloud controller manager for {alibaba}
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|Removed
```

Comment:
Same with https://github.com/openshift/openshift-docs/pull/80829#discussion_r1734096067

Reply:
see https://github.com/openshift/openshift-docs/pull/80829#discussion_r1734732263

Thanks for bringing this up! It highlighted a discrepancy in the RN tables that we will fix as a result :nerd_face:

Reply:
Good to know, thank you Jeana!
/lgtm